frontend/src/hooks/useStore.ts

```diff
@@ -92,4 +88,14 @@ export const useStore = create<State>((set) => ({
             .then(() => set({isLoading: false}));
 
     },
+
+    selectCard: (card: CardType) => {
+        set({selectedCard: card});
+        console.log(card);
```

Comment:
Forgot to remove or do you need it?
Nevermind. Just wanted to leave a comment in your code xD
```suggestion

```

Reply:
this is just so i can control the card values when clicking. Was pretty helpful so far, will remove it in the future :D